Formula/elasticsearch.rb

```diff
@@ -21,10 +27,11 @@ def cluster_name
   def install
     if build.head?
       # Build the package from source
-      system "gradle", "clean", "assemble"
-      # Extract the package to the current directory
-      targz = Dir["distribution/tar/build/distributions/elasticsearch-*.tar.gz"].first
-      system "tar", "--strip-components=1", "-xf", targz
+      system "gradle", "clean", ":distribution:tar:assemble"
+      # Extract the package to the tar directory
+      mkdir "tar"
```

Comment:
Apologies, the compressed diff made that difficult to spot. It's alright as-is then.